### DISCUSSION TITLE
Working with tenda w311mi

### DISCUSSION THREAD

**Discussion by preetbhatti** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#62] [Show and tell]
This is life saver for me, just needed to downloaded **Wireless.USB.OC.Big.Sur.Adapter-V7** and finish the setup steps.
That is all, restarted the system and The icon with usb adapter wifi appears in top bar.
![Screenshot 2021-06-19 at 6 13 12 PM](https://user-images.githubusercontent.com/35762470/122642850-24226e80-d12a-11eb-96a8-3fe0e289a8ac.png)

**Comment by chris1111** [comment]
Thanks  I will add to the working list adapter 😃

**Reply by Wired-cell** [reply]
I have the same adapter but it's not working on big sur. Any solution to this problem ? Also, does it have additional requirements like mapping usb ports ? (I use usbinjectall )

**Comment by DERMOIIN** [comment]
> This is life saver for me, just needed to downloaded **Wireless.USB.OC.Big.Sur.Adapter-V7** and finish the setup steps. That is all, restarted the system and The icon with usb adapter wifi appears in top bar. ![Screenshot 2021-06-19 at 6 13 12 PM](https://user-images.githubusercontent.com/35762470/122642850-24226e80-d12a-11eb-96a8-3fe0e289a8ac.png)
why its not working for me?
can you assist me?
